hack/ghub-set-vars

```diff
@@ -1,45 +1,67 @@
 #!/bin/bash
-SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
 
-if [ $# -eq 1 ]; then
-    REPO_URL=$1
-    # Assume $REPO_URL is like "https://github.com/org/repo"
-    # and use cut to pull out the "org/repo" part
-    ORG_AND_REPO=$(echo "$REPO_URL" | cut -d/ -f4,5)
+# Helper script used to simplify setting
+# variables and secrets in a GitHub repository
+
+set -eou pipefail
```

Comment:
I think this needs to be `-euo pipefail` because `pipefail` is part of the `-o flag`.